### DISCUSSION TITLE
Sub folder sizes and is this same as Xplorer from zabkat

### DISCUSSION THREAD

**Discussion by gl2007** [kimlimjustin/xplorer#245] [Ideas]
Is this same as the xplorer² Windows File Manager [https://www.zabkat.com](https://www.zabkat.com/] ?

I use the size option of subfolder quite a lot from the xplorer2 which I linked. 

If this is not the same, may I suggest that feature? Helps a lot while dealing with disk storage full issues.

**Comment by kimlimjustin** [comment]
> Is this same as the xplorer² Windows File Manager [https://www.zabkat.com]([https://www.zabkat.com/]](https://www.zabkat.com/%5D) ?

Nope, it's different.

> I use the size option of subfolder quite a lot from the xplorer2 which I linked.
> If this is not the same, may I suggest that feature? Helps a lot while dealing with disk storage full issues.

Could you please elaborate on this?

**Comment by gl2007** [comment]
What I meant is that in xplorer2, there is a column for Size (not sure if
it is same as the one for files or if they have a separate column for
folder) which shows the full recursive size for sub Folders. It calculates
this in the background but has to be enabled by a shortcut or menu item
since it takes time to calculate it. But maybe keep a hidden file with
cached last known size and last modified date, and change only if any
updates are detected in subfolder (couldnt resist solutioning).

**Reply by kimlimjustin** [reply]
Delivered to you by #246, you may try it locally and give it a review and/or request any changes you'd like. It's completely OK if you can't do so, I'll merge this the day after tomorrow if no changes are requested.

**Comment by gl2007** [comment]
Thank you Justin. I will be able to test out only Monday. But please go
ahead with the merge. I appreciate your quick response and implementation!!

Regards,
Ghanshyam
